less/variables.less

```diff
@@ -1,14 +1,20 @@
 
+// Colors
+@white: #FFFFFF;
+@silver: #C0C0C0;
+@shark: #212529;
+@eastern-blue: #17A2B8;
+@sandy-beach: #FFEBCD;
+
 // Color assignments 
 @font-color: @shark;
 @button-bg: @eastern-blue;
 @main-bg:  @white;
 @navigation-border: @silver;
 @footer-bg: @sandy-beach;
 
-// Colors
-@white: #FFFFFF;
-@silver: #C0C0C0;
-@shark: #212529;
-@eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+
+
+// media size
+@mobile-width: 500px;
```

Comment:
While this is a valid way of setting the width, the README asked you to use escaping.